### DISCUSSION TITLE
Spaceman is not allowed to send keystrokes

### DISCUSSION THREAD

**Discussion by chianyungcode** [ruittenb/Spaceman#4] [Q&A] [answered]
Hi i got problem with the apps, already set the accessibility on in my settings. I'm on latest version 1.6.19

![CleanShot 2024-12-31 at 14 47 36@2x](https://github.com/user-attachments/assets/69c06935-8263-470a-aa71-7f2f23bfa9bc)

<img width="642" alt="image" src="https://github.com/user-attachments/assets/d14140db-a03d-4d5e-8664-4471f16ed1c3" />

**Comment by ruittenb** [comment]
Hello @chianyungcode, sometimes it is necessary to remove the old version from the permissions list and add the new version of the application explicitly. Could you try that?

**Comment by ruittenb** [comment, marked as answer]
@chianyungcode this is what I mean: remove and re-add Spaceman here:

<img width="715" alt="remove-then-add" src="https://github.com/user-attachments/assets/893a0f38-60d8-4def-9257-d34e56fe5fae" />
